### PR TITLE
[Add-ins] (Mac) Reference saveAsync workaround

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[43439d6](https://github.com/OfficeDev/outlook-dev-docs/pull/596/commits/43439d6ec852f75c50ae5929901618edf49c0b50)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-mac-saveasync/docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C29%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201905292236250025-596%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=we5zQx2nq5CBh2gq7U6vGwm3ctM7EUCpHC969GhhIec%3D&st=2019-05-29T22%3A32%3A30Z&se=2019-06-29T22%3A37%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[32e4e32](https://github.com/OfficeDev/outlook-dev-docs/pull/596/commits/32e4e32130f4dad6b83ca8fa21ebb1ccf8ba3e25)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-mac-saveasync/docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201905301748559204-596%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=LeOHUlO6AK23OsDQ2WYeI66yoi2jIFTCmdoUf7gxdT8%3D&st=2019-05-30T17%3A44%3A51Z&se=2019-06-30T17%3A49%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ For more information, see [Deploy and install Outlook add-ins for testing](testi
 
 | Area | Outlook on Windows, OWA for Devices, Outlook on the web | Outlook for Mac |
 |:-----|:-----|:-----|
-| Supported versions of office.js and Office Add-ins manifest schema | All APIs in Office.js and schema v1.1. | <ul><li>Supports requirement sets 1.0, 1.1, 1.2, 1.3, and 1.4</li><li>Schema v1.1.</li></ul>**NOTE**: Outlook for Mac does not support saving a meeting. The `saveAsync` method will fail when called from a meeting in compose mode. |
+| Supported versions of office.js and Office Add-ins manifest schema | All APIs in Office.js and schema v1.1. | <ul><li>Supports requirement sets 1.0, 1.1, 1.2, 1.3, and 1.4</li><li>Schema v1.1.</li></ul>**NOTE**: Outlook for Mac does not support saving a meeting. The `saveAsync` method will fail when called from a meeting in compose mode. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround. |
```
Comment (AlexJerabek) [inline docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md]: Consider "method fails" instead of "method will fail".

Comment (ElizabethSamuel-MSFT) [inline docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md]:
```suggestion
| Supported versions of office.js and Office Add-ins manifest schema | All APIs in Office.js and schema v1.1. | <ul><li>Supports requirement sets 1.0, 1.1, 1.2, 1.3, and 1.4</li><li>Schema v1.1.</li></ul>**NOTE**: Outlook for Mac does not support saving a meeting. The `saveAsync` method fails when called from a meeting in compose mode. See [Cannot save a meeting as a draft in Outlook for Mac by using Office JS API](https://support.microsoft.com/help/4505745) for a workaround. |
```

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[0c86729](https://github.com/OfficeDev/outlook-dev-docs/pull/596/commits/0c867291b5bd6900d732dd3ac2e654f846e396d7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md](https://github.com/OfficeDev/outlook-dev-docs/blob/elizs-mac-saveasync/docs/add-ins/compare-outlook-add-in-support-in-outlook-for-mac.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5Cf49385f6-041e-c905-242e-e0025c631156%5CPullRequest%5C201905301934108521-596%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=l6YjCHzmUFQB2O7Jjo1HPabXKhsKlFwIXOdcnAnvPy4%3D&st=2019-05-30T19%3A30%3A17Z&se=2019-06-30T19%3A35%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**